pkg/feature/templates/serverless/serving-install/knative-serving.tmpl

```diff
@@ -21,3 +21,5 @@ spec:
   config:
     istio:
       local-gateway.knative-serving.knative-local-gateway: "knative-local-gateway.{{ .ControlPlane.Namespace }}.svc.cluster.local"
+    network:
+      domainTemplate: "{{.Name}}.{{.Namespace}}.{{.Domain}}"
```

Comment:
I defer to @retocode, @skonto and/or @rhuss for a review.
They are the right people to tell if this change is still under support.

Reply:
Hm why would you want to make this configurable in the first place? 
In theory this is (or could be) supported, but as we have an additional layer of routing in OpenShift which also needs to match the configuration we have in Knative. Depending on what people put there, OCP routing could stop working, so I'd vote to not change it. 

Btw, the flag would be `domain-template` using the newer syntax.

Reply:
AFaik we changed the default downstream in order to align with OCP [conventions](https://issues.redhat.com/browse/SRVKS-56).